src/index.ts

```diff
@@ -52,12 +52,14 @@ class Transformer {
     }
 
     private _applyEdits(file: VinylFile, edits: ts.TextChange[]): Buffer {
-        return edits.reduceRight((contents, edit) => {
-            var head = contents.slice(0, edit.span.start());
-            var tail = contents.slice(edit.span.start() + edit.span.length());
+        const reduce = (contents, edit) => {
```

Comment:
Couldn't figure out how to make linter happy (see https://github.com/palantir/tslint/issues/742)